### PR TITLE
feat(runtime): add live audit timeline to local dashboard

### PR DESCRIPTION
**Pull request by XiaokunDuan**
## Problem

The local dashboard can show an empty Event History even while a local takeover is actively running. The supervisor event bus only captures structured runtime events, so agent process stdout/stderr and local supervisor activity can be invisible. In practice this makes a healthy long-running takeover look idle or stale.

## Behavior

This PR adds a unified read-only audit timeline to the local dashboard.

- Persist local supervisor stdout/stderr to `.wanman/runtime-audit.log`.
- Read runtime audit logs alongside supervisor event-bus history.
- Add `/dashboard/events` for live dashboard SSE updates.
- Render runtime audit entries, event-bus entries, and legacy audit lines in one Event History timeline.
- Preserve raw JSON/raw line content for debugging.
- Use browser-local 24-hour timestamps and keep the timeline pinned to the newest entry unless the user scrolls up.

Agent scheduling, task execution, and decision logic are unchanged.

## Screenshots

Live dashboard overview with runtime state and unified Event History:

Live runtime audit dashboard

Browser verification screenshot from the local dashboard:

Browser-verified dashboard timeline

## Tests

Full suite:
- `pnpm typecheck` — 4 package typecheck tasks passed.
- `pnpm test` — full Turbo test suite passed.

Focused coverage:
- `@wanman/runtime`: dashboard data reads runtime audit logs and keeps raw audit payloads.
- `@wanman/runtime`: `/dashboard/events` streams live dashboard events.
- `@wanman/cli`: local supervisor stdout/stderr is persisted to `.wanman/runtime-audit.log`.

Manual verification:
- Ran a local takeover and verified the dashboard in Chrome / Computer Use.

### CODE REVIEW

File: packages/runtime/src/supervisor.ts [hunk 1]
```diff
@@ -1942,6 +2052,207 @@ ${activePaths}`;
...
+    const candidates = overlayRoot
+      ? [
+          { path: path.join(overlayRoot, 'runtime-audit.log'), source: 'runtime-audit' as const },
+          { path: path.join(overlayRoot, 'epistemic-takeover.log'), source: 'legacy-audit' as const },
+        ]
+      : [];
+
+    for (const candidate of candidates) {
+      if (!fs.existsSync(candidate.path)) continue;
+      const raw = fs.readFileSync(candidate.path, 'utf-8');
```
Comment (guo-yu) [inline packages/runtime/src/supervisor.ts:2189]: This endpoint now reads and parses the entire `runtime-audit.log` synchronously on every `/dashboard/data` request, and the dashboard polls that endpoint every 3 seconds. In a long-running local takeover, supervisor stdout/stderr can grow without bound, so this can block the supervisor HTTP event loop and inflate every response payload. Please cap this path, for example by tailing the last N bytes/lines, and also cap the in-memory `dashboardEvents` history so the dashboard remains bounded over long runs.

File: packages/runtime/src/supervisor.ts [hunk 2]
```diff
@@ -1942,6 +2052,207 @@ ${activePaths}`;
...
+    healthChecks: Array<{ label: string; status: string }>;
+  } {
+    const health = this.getHealth();
+    const tasks = this.taskPool.listSync();
+    const initiatives = this.initiativeBoard.listSync();
+    const capsules = this.capsulePool.listSync();
+    const artifacts = this.listDashboardArtifacts();
+    const audit = this.readRuntimeAuditTrail();
+    const live = this.readLiveDashboard();
+    const events = [...audit.entries, ...this.dashboardEvents];
```
Comment (guo-yu) [inline packages/runtime/src/supervisor.ts:2081]: The Event History is described as a unified timeline, but this concatenates all audit-log entries before all event-bus entries. That means a newer runtime-audit line can appear before an older event-bus entry, which breaks the chronological timeline and the pinned-newest behavior in the browser. Please merge by timestamp/sequence, or maintain a single bounded append-only timeline that preserves actual event order.

**Review by guo-yu — COMMENTED** [review]
Thanks for the detailed implementation and screenshots. I verified the latest head locally: focused runtime/CLI tests, full `pnpm test`, and `pnpm typecheck` all pass. I would hold merge for the two dashboard-data issues below.

File: packages/runtime/src/supervisor.ts (hunk 1 above)
Comment (XiaokunDuan) [reply packages/runtime/src/supervisor.ts:2189]: Certainly, I will test more on presure load and some edge cases.
